src/components/GiveProject/GrantCard.tsx

```diff
@@ -318,37 +268,7 @@ export default function GrantCard({ grant, mode }: GrantDetailsProps) {
       return dispatch(error(t`Please enter a value!`));
     }
 
-    // If on Rinkeby and using Mock Sohm, use changeMockGive async thunk
-    // Else use standard call
-    if (networkId === NetworkId.TESTNET_RINKEBY && Environment.isMockSohmEnabled(location.search)) {
```

Comment:
I don't see any use of the mock sOHM contract in the new implementation. Do we really want to remove it?